README.md

```diff
@@ -1,7 +1,135 @@
+
+This will open a Cloud Shell window and clone a copy of this repository. Cloud
+Shell has your credentials built in, and Terraform is already installed, so
+you're ready to move on to running the demo. You can also [open Cloud Shell the normal way](https://cloud.google.com/shell/docs/using-cloud-shell)
+but you'll need to clone this repo yourself.
+
+If you want to work locally, you'll need to [install the `gcloud` tool](https://cloud.google.com/sdk/docs/install).
+Confidential to Mac users: `gcloud` is also available [via homebrew](https://formulae.brew.sh/cask/google-cloud-sdk).
+Once the install is done, run `gcloud init` to set up your environment. More
+information can be found [in the docs](https://cloud.google.com/sdk/docs/initializing).
+
+### Installing Terraform
+
+If you're using Cloud Shell, you've already got Terraform installed so feel free
+to move on to the next section. Otherwise, [download Terraform](https://www.terraform.io/downloads)
+and install it. A detailed walkthrough can be found [in this tutorial](https://learn.hashicorp.com/tutorials/terraform/install-cli).
+
+### Choosing a region
+
+Next, select a region for your infrastructure. For this demo it's okay to choose
+somewhere close to you, but for production use you may want to choose a different
+region. If you need some advice, check out [Best Practices for Compute Engine regions selection](https://cloud.google.com/solutions/best-practices-compute-engine-region-selection).
+
+For a list of available regions, refer Atlas's [Google Cloud provider documentation](https://www.mongodb.com/docs/atlas/reference/google-gcp/).
+Choose a region that's close to you and that supports the `M0` cluster tier. Make
+a note of both the Google Cloud region name and the Atlas region name, as you'll
+need both in the next step.
+
+### Configuring the demo
+
+If you haven't already, clone this repo. Run `terraform init` to make sure
+Terraform is working correctly and download the provider plugins.
+
+Then, create a file in the root of the repository called `terraform.tfvars` with
+the following contents, replacing placeholders as necessary:
+
+    atlas_pub_key        = "<your Atlas public key>"
+    atlas_priv_key       = "<your Atlas private key>"
+    atlas_org_id         = "<your Atlas organization ID>"
+
+If you selected the `us-central1`/`US_CENTRAL` region then you're ready to go. If
+you selected a different region, add the following to your `terraform.tfvars` file:
+
+    atlas_cluster_region = "<Atlas region ID>"
+    google_cloud_region  = "<Google Cloud region ID>"
+
+Run `terraform init` again to make sure there are no new errors. If you get an
+error, check your `terraform.tfvars` file.
+
+### Run it!
+
+You're finally ready to deploy your application. Run `terraform apply` and you
```

Comment:
Best practice is usually to run a `plan` first as it completely eliminates the possibility of accidentally deploying a mistake, but I guess since this is a demo, the single prompt in `apply` is sufficient. Maybe adding a little blurb that says something like "If you want to do a dry-run, use `terraform plan`. When you're ready to actually deploy everything, run `terraform apply`"

Reply:
added a note about `terraform plan` in 6b8cabf.